KoreBuild-dotnet/build/_k-standard-goals.shade

```diff
@@ -111,7 +112,14 @@ default NUGET_FEED = 'https://api.nuget.org/v3/index.json'
       .Where(f => Path.GetFileName(f).Equals("project.json", StringComparison.OrdinalIgnoreCase))
       .ToList();
 
-    projectFiles.ForEach(projectFile => DotnetBuild(projectFile, Configuration));
+    projectFiles.ForEach(projectFile =>
+      {
+        var projectName = Path.GetFileName(Path.GetDirectoryName(projectFile));
+        foreach (var framework in GetFrameworks(projectFile))
+        {
+          DotnetPublish(projectFile, Path.Combine(TEST_BUILD_DIR, DefaultRid, framework, projectName), framework, Configuration);
```

Comment:
Yeah, that's probably a more reasonable approach. I'll rip that out. Is that the only concern holding back this change?